kubernetes/assets/dashboards/kubernetes_jobs.json

```
@@ -0,0 +1,1470 @@
+                                    "order": "desc",
+                                    "q": "top(sum:kubernetes_state.job.count{$cluster_name} by {kube_cluster_name}, 25, 'last', 'desc')"
+                                },
+                                {
+                                    "aggregator": "avg",
+                                    "alias": "Average number of jobs",
+                                    "q": "top(sum:kubernetes_state.job.count{$cluster_name} by {kube_cluster_name}, 25, 'last', 'desc')"
+                                }
+                            ],
+                            "time": {},
+                            "title": "Number of Jobs per cluster",
+                            "title_align": "left",
+                            "title_size": "16",
+                            "type": "query_table"
+                        },
+                        "id": 7493103153404182,
+                        "layout": {
+                            "height": 3,
+                            "width": 5,
+                            "x": 0,
+                            "y": 8
+                        }
+                    },
+                    {
+                        "definition": {
+                            "legend_columns": [
+                                "avg",
+                                "min",
+                                "max",
+                                "value",
+                                "sum"
+                            ],
+                            "legend_layout": "auto",
+                            "markers": [],
+                            "requests": [
+                                {
+                                    "display_type": "line",
+                                    "formulas": [
+                                        {
+                                            "alias": "Jobs",
+                                            "formula": "exclude_null(query1)"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "sum:kubernetes_state.job.count{$cluster_name} by {kube_cluster_name}"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "style": {
+                                        "line_type": "solid",
+                                        "line_width": "normal",
+                                        "palette": "dog_classic"
+                                    }
+                                }
+                            ],
+                            "show_legend": false,
```

```suggestion
                            "show_legend": true,
```

Enabling the legend as the widget is big enough in order to increase readability and being consistent with the widget below